src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1471,5 +1471,6 @@ internal enum ErrorCode
         ERR_InvalidPreprocessingSymbol = 8301,
         ERR_FeatureNotAvailableInVersion7_1 = 8302,
         ERR_LanguageVersionCannotHaveLeadingZeroes = 8303,
+        ERR_DuplicateBoundWithDifferentTupleNames = 8304,
```

Comment:
>ERR_DuplicateBoundWithDifferentTupleNames = 8304 [](start = 8, length = 48)

Is this going to conflict with an error for a new feature? Perhaps we can use the gap between 8209 and 8300 for bug fixes?